maker/src/routes.rs

```diff
@@ -45,8 +45,8 @@ pub async fn get_offer(
 
     match quote {
         Some(quote) => Ok(Json(Offer {
-            bid: (quote.ask * (Decimal::ONE + spread)),
-            ask: (quote.bid * (Decimal::ONE - spread)),
+            bid: (quote.bid * (Decimal::ONE - spread)),
+            ask: (quote.ask * (Decimal::ONE + spread)),
```

Comment:
Yeah, we're _bidding less_ when the taker sells to us and _asking for more_ when they buy from us.